LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/align-content-001.htm

```diff
@@ -7,7 +7,7 @@
         <link rel="help" href="http://www.w3.org/TR/css-flexbox-1/#align-content-property" />
         <link rel="match" href="reference/align-content-001-ref.html" />
         <meta name="assert" content="This test checks that a multi-line flex container with 'align-content: center' centers lines in the flex container." />
-        <meta name="fuzzy" content="maxDifference=0-1; totalPixels=0-5709" />
+        <meta name="fuzzy" content="maxDifference=1; totalPixels=6217" />
```

Comment:
If you don't use `0-*` ranges, it means you expect _exactly_ a certain maxDifference/totalPixels, and any other number (even lower), will cause the test to fail.

This is not great in case we progress, but also not accurate since other browsers/platforms may have different fuzziness.

```suggestion
        <meta name="fuzzy" content="maxDifference=0-1; totalPixels=0-6217" />
```

Reply:
I see, thanks Tim! I think it should be better now. Let me know what you think.